bindings/python/tests/test_postgresql1_write.py

```diff
@@ -54,14 +54,14 @@ def ping_server(server="localhost", port=5432, timeout=3):
 ping_server()
 
 # Add metadata to search path
-dlite.storage_path.append(f"{thisdir}/Person.json")
+dlite.storage_path.append(f"{thisdir}/*Person.json")
```

Comment:
This being a test and all, I would add both files explicitly instead of using a wildcard. If you want to test the wildcard append, create a different test

Reply:
Done. The wildcard and even just specifying the directory are tested elsewhere.